external/boost/archive/portable_binary_archive.hpp

```diff
@@ -37,7 +37,8 @@ enum portable_binary_archive_flags {
 //#endif
 
 inline void
-reverse_bytes(char size, char *address){
+reverse_bytes(signed char size, char *address){
+    BOOST_ASSERT(size > 0);

```

Comment:
But in both of [iarchive](https://github.com/monero-project/monero/blob/dd580d7bc751018017e61f4852f68896f6edf620/external/boost/archive/portable_binary_iarchive.hpp#L241) and [oarchive](https://github.com/monero-project/monero/blob/dd580d7bc751018017e61f4852f68896f6edf620/external/boost/archive/portable_binary_oarchive.hpp#L230), the case where `size==0` is clearly excluded. Should I still use >= ?